library/Icingadb/Model/Hostgroupsummary.php

```diff
@@ -97,22 +99,6 @@ public function getDefaultSort()
     public function getUnions()
     {
         $unions = [
-            [
-                Hostgroup::class,
```

Comment:
Oh no, why have you done this? Is this really required? This will break cases of `getUnions()[0]`.